src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import core.basesyntax.db.DaoService;
+import core.basesyntax.db.impl.DaoServiceImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.Calculator;
+import core.basesyntax.service.CsvFileReaderService;
+import core.basesyntax.service.CsvFileWriterService;
+import core.basesyntax.service.DataProcessor;
+import core.basesyntax.service.impl.CalculatorImpl;
+import core.basesyntax.service.impl.CsvFileReaderServiceImpl;
+import core.basesyntax.service.impl.CsvFileWriterServiceImpl;
+import core.basesyntax.service.impl.DataProcessorImpl;
+import java.util.List;
+
+public class Application {
+    private static final String READ_FROM_FILE =
+            "src\\main\\java\\core\\basesyntax\\resourses\\ValidReadFromFile.csv";
+    private static final String NEGATIVE_QUANTITY_READ_FROM_FILE =
+            "src\\main\\java\\core\\basesyntax\\resourses\\NegativeQuantityReadFromFile.csv";
+    private static final String WRITE_TO_FILE =
+            "src\\main\\java\\core\\basesyntax\\resourses\\WriteToFile.csv";

```

Comment:
as described in common mistakes, please put input and output files in src/main/resources. Also consider making names more clear